docs/getting-started/integrations/angular.md

```diff
@@ -98,7 +92,7 @@ npm install --save @ckeditor/ckeditor5-angular
 	If you do not have an existing project, you can use the [Angular CLI](https://angular.io/cli) to create a new one.
 </info-box>
 
-Install one of the {@link getting-started/legacy-getting-started/predefined-builds CKEditor&nbsp;5 predefined builds} or [create a custom one](#using-a-custom-ckeditor-5-build).
+Install one of the {@link getting-started/legacy-getting-started/predefined-builds CKEditor&nbsp;5 predefined builds}<!-- well this definitely needs some love --> or get one from [Builder](https://ckeditor.com/builder?redirect=docs).
```

Comment:
Can we safely get rid of the first part of this sentence?

Reply:
I'm working on this integration now. You can take a look after I'll push it.